### ISSUE TITLE
dev/emc: follow-up changes for PR #330

### ISSUE THREAD

**Issue by climbfuji** [NOAA-GFDL/GFDL_atmos_cubed_sphere#345]
**Is your feature request related to a problem? Please describe.**

During the review process of https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/330, the following items were noted as follow-up work:
- in `driver/fvGFS/atmosphere.F90`, the logic around lines 2152 is redundant - see https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/330/files/85faba53a23060b301c898a438d3df2c44fb14a3#diff-53ee59c906ed74739c938e8c964c9c4fc6b71627e5cd6a5741e2cdacedfc16ab for details
- in `driver/fvGFS/atmosphere.F90`, remove comment in line 1615
- revisit the files change in PR #330 and remove all unnecessary block loops (`do nb=1,blen ...`)

**Describe the solution you'd like**

See above

**Describe alternatives you've considered**

Leave things as is - not a good idea, since we will be missing out on expected performance improvements

**Additional context**

n/a

**Comment by bensonr** [comment]
@climbfuji - did this get addressed in #330 or is it a follow on issue to be addressed?

**Comment by climbfuji** [comment]
No, this is indeed follow-up work that will hopefully lead to less code and small performance gains. Hopefully someone from EMC can pick this up.

**Comment by bensonr** [comment]
Thanks, we will leave this open.
